draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3634,6 +3634,21 @@ Upgrade: websocket
    (<xref target="attack.protocol.element.length"/>).
 </t>
 <t>
+   Because it is used for message delimitation in HTTP/1.1, the Content-Length
+   header field value can affect the security of the protocol. This is true even
+   when the current connection is not using HTTP/1.1, because HTTP is an
+   intermediated protocol, and messages might be transmitted using other versions
+   of the protocol.
```

Comment:
```suggestion
   Because Content-Length is used for message delimitation in HTTP/1.1,
   its field value can impact how the message is parsed by downstream
   recipients even when the immediate connection is not using HTTP/1.1.
   If the message is forwarded by a downstream intermediary, a Content-Length
   field value that is inconsistent with the received message framing might
   cause a security failure due to request smuggling or response splitting.
```